### PR TITLE
Umcs 363/change/add unzer logo to the project and link it in readme file

### PR DESCRIPTION
**Pull request by DerAusHeidelberg**

### CODE REVIEW

**Review by sixer1182 — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres
 to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.3.0][1.1.3.0]
+
+### Changed
+
+*   Added Unzer-Logo and updated the reference in Readme-File
+*   Added Json-Validation to JsonParser-Class before real Json-Deserialization happens
+*   Added Twitter-Handle to Readme-File
```
Comment (Ryouzanpaku) [inline CHANGELOG.md:14]: typo: I would put points at the end of those sentences.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres
 to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.3.0][1.1.3.0]
```
Comment (Ryouzanpaku) [inline CHANGELOG.md:8]: question: Based on the contained changes so far, wouldn't it be version "1.1.2.6" instead?

**Review by Ryouzanpaku — DISMISSED** [review]
LGTM
but check comments pls.

File: CHANGELOG.md (hunk 2 above)
Comment (sixer1182) [reply CHANGELOG.md:8]: You are right.
I changed it in the Jira release.

**Review by Ryouzanpaku — DISMISSED** [review]
LGTM.

**Review by Ryouzanpaku — APPROVED** [review]

**Pull request merged**